src/UI/Factory.php

```diff
@@ -336,4 +337,27 @@ public function panel();
 	 **/
 	public function modal();
 
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *     Popovers are used to display secondary information to a component.
```

Comment:
Sorry I did not update the descriptions of the factory. Thanks for your suggestions, they will be added with the next commits.